rest-api/dao/biobank_order_dao.py

```diff
@@ -174,6 +174,13 @@ def get_with_children(self, obj_id):
     with self.session() as session:
       return self.get_with_children_in_session(session, obj_id)
 
+  def get_biobank_order_for_participant(self, pid):
```

Comment:
can we rename this to the plural `..._orders_...`

Reply:
done